packages/segment/src/event-helpers.ts

```diff
@@ -87,3 +87,13 @@ export const identifyUser = (
     options,
   };
 };
+
```

Comment:
👇 A helper doesn't _have_ to accept an event definition as an argument. This can be reduced to:

```suggestion
export const reset = (): EventDefinition => () => ({
  hitType: 'reset',
});
```

Reply:
ah I thought it need to call the eventdef. I'll update